### ISSUE TITLE
[Feature request] Audiodelay adjustment is way too coarse

### ISSUE THREAD

**Issue by haraldbacklund** [iina/iina#2626]
- macOS 10.14.6
- IINA 1.0.4 Build 107

Audio- and subdelay values (using keyboard shortcut or menu option) are way too coarse at 0.5s. Should be 0.05 or at least 0.1sec. There is a box for inputing custom values but that is a very unpractical way of finding a good value, since you have to use sliders.

This was an issue [over a year ago](https://github.com/iina/iina/issues/1758) when it was inexplicably closed by someone with the response "just use the custom value box". 

An alternative would be to allow some sort of value preference to be set for the keyboard shortcuts/slider, but I think that would add unnecessary complexity.

I see very few benefiting from having the value set this high and very little harm being made compared to the use of a change.

Tried to fork and fix it myself but couldn't make sense of where the values were set.

**Comment by clankill3r** [comment]
Yeah 0.5 sec is insane. 0.1 or 0.2 would be fine.

**Comment by nielsboecker** [comment]
Agreed, it would be very reasonable to have a finer granularity for audio adjustments to be changed conveniently with the keyboard shortcuts, similar to how VLC provides it (VLC shifts in 50 ms steps).

**Comment by haakonstorm** [comment]
I made this glorious GIF to explain the process.
It features [a 60 year old who kills hip hop](https://www.youtube.com/watch?v=M-MqpaAU_PM).
![iina_delay](https://user-images.githubusercontent.com/193781/110721565-15feb100-8211-11eb-9223-973f4f9211e2.gif)

**Comment by haakonstorm** [comment]
*(This issue should be closed as the functionality requested has existed for years – as shown in my GIF animation above (albeit not exactly easy to find!))*
